rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
Rather than deal with unknown (ansible?) bugs giving an inconsistent experience (I'm not recreating this, and neither @nrb nor @prometheanfire seem to have encoutered it before) I'm going to just fold it into the user_extra_variables for right now and move on.